src/lib/components/MainPanel.js

```diff
@@ -68,6 +68,6 @@ export default class MainPanel extends React.Component {
   confirmTimeSelect(dateChoice) {
     const { scheduleSnoozedTab } = this.props;
     if (!dateChoice) { return; }
-    scheduleSnoozedTab(dateChoice);
+    scheduleSnoozedTab(dateChoice, 'pick');
```

Comment:
I think we should use the constant from `lib/times.js` here instead.  (And add one for `'week'` down below.)